metadata-ingestion/docs/sources/airbyte/airbyte_pre.md

```diff
@@ -0,0 +1,16 @@
+## Configuration Notes
+1. Airbyte source is available for both Airbyte Cloud and Airbyte Open Source (OSS) users.
+2. For Airbyte Cloud user need to provide api_key in recipe to ingest metadata and for Airbyte OSS username and password.
```

Comment:
Airbyte Cloud users need to provide `api_key` in the recipe for authentication and Airbyte OSS users need to provide `username` and `password`.

Reply:
Done